### PR TITLE
WIP - create a database handling trait and implements it for postgresql connection

### PR DESCRIPTION
**Pull request by jean553**
This is the first PR for:
https://github.com/allan-simon/sentence-aligner-rust/issues/52

### CODE REVIEW

File: tests/test_create_language.rs [hunk 1]
```diff
@@ -35,13 +41,7 @@ fn test_post_language_returns_200() {
         StatusCode::Created,
     );
 
-    assert!(
-        db::language_exists(
-            &connection,
-            &created_language,
-        ),
-        "The language has not been inserted."
```
Comment (allan-simon) [inline tests/test_create_language.rs]: we've lost this error message in the transition

File: tests/test_create_language.rs [hunk 2]
```diff
@@ -14,8 +21,7 @@ mod tests_commons;
 #[test]
 fn test_post_language_returns_200() {
 
-    let connection = db::get_connection();
-    db::clear(&connection);
+    let connection: Connection = DatabaseHandler::connect_and_clean();
```
Comment (allan-simon) [inline tests/test_create_language.rs]: is the `: Connection` necessary ?

Comment (jean553) [reply tests/test_create_language.rs]: It surprises me too, but yes.

File: tests/test_create_language.rs (hunk 1 above)
Comment (jean553) [reply tests/test_create_language.rs]: Done

**Pull request merged**